pywbem_mock/_baseprovider.py

```diff
@@ -79,55 +63,20 @@ class BaseProvider(object):
     executed directly.
```

Comment:
In the docstring of class `BaseProvider` it references class `MainProvider` which is not shown in the docs. I suggest to change this to: "... required by built-in providers and by user-defined providers."

Reply:
DONE